physx/Cargo.toml

```diff
@@ -18,7 +18,7 @@ physx-sys = { version = "0.4.10", path = "../physx-sys" }
 
 enumflags2 = "0.6"
 log = "0.4"
-glam = "0.12"
+glam = { version = "0.13", optional = true }
```

Comment:
One could also support a range of glam releases here, and update that more frequently. It is harder to test though and CI should be on the oldest version. But it is another parallel path forward as well